tests/subsys/pm/power_mgmt/boards/native_posix.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	deviceA: deviceA {
+		label = "deviceA";
+		compatible = "device-a";
```

Comment:
does `device-a` compatible exist? better create a dummy compatible in `test` bindings folder, e.g. `vnd,pm.yaml`.

Reply:
I'll be consistent with other tests and use "test-device-pm". Following:

`test-gpio-basic-api`
`test-regulator-fixed`
`test-pwm-loopback`